packages/bpk-docs/src/pages/NativeBadgePage/NativeBadgePage.js

```diff
@@ -51,6 +55,50 @@ const components = [
       },
     ],
   },
+  {
+    id: 'with-icon',
+    title: 'With an icon',
+    screenshots: [
+      {
+        title: 'iOS',
+        width: 750,
+        height: 1334,
+        src: `/${iosScreenshotWithIcon}`,
+        altText: 'iOS Badge Component with a single icon',
+        subText: '(iPhone 8 simulator)',
+      },
+      {
+        title: 'Android',
+        width: 1080,
+        height: 1920,
+        src: `/${androidScreenshotWithIcon}`,
+        altText: 'Android Badge Component with a single icon',
+        subText: '(Google Pixel emulator)',
+      },
+    ],
+  },
+  {
+    id: 'with-icons',
+    title: 'With multiple icons',
```

Comment:
The icon separator is optional, so let's make it clear it's been supplied here.
```suggestion
    title: 'With multiple icons and an icon separator',
```